### DISCUSSION TITLE
Nowość: tryb ciemny w RadioLiście!

### DISCUSSION THREAD

**Discussion by TomaszGasior** [TomaszGasior/RadioLista-v3#231] [0. Ogłoszenia Announcements]
Na radiolista.pl wdrożyłem właśnie nową wersję aplikacji. Najważniejszą nowością jest opcja trybu ciemnego, ciemnej wersji kolorystycznej serwisu idealnej do korzystania z RadioListy późnym wieczorem lub w nocy. Tryb ciemny można włączyć ręcznie z poziomu stopki strony, ale integruje się tez on w schematem kolorystycznym ustawionym w systemie operacyjnym. Tryb ciemny jest globalny, dotyczy całego serwisu i można go sobie ustawić również jako użytkownik niezalogowany. https://github.com/TomaszGasior/RadioLista-v3/issues/203

W ramach [porządków](https://github.com/TomaszGasior/RadioLista-v3/milestone/2?closed=1) w kodzie w RadioListy została usunięta możliwość personalizowania kolorów w wyglądzie poszczególnych wykazów. Uzasadnienie tej decyzji znajduje się w zadaniu https://github.com/TomaszGasior/RadioLista-v3/issues/202.

W ustawieniach wykazu i ustawieniach profilu z powrotem zaczęło działać pole edycji opisu. Przez ostatnie kilka tygodniu w wyniku błędu nie działało prawidłowo. https://github.com/TomaszGasior/RadioLista-v3/issues/229 https://github.com/TomaszGasior/RadioLista-v3/issues/221

Będę wdzięczny za informację zwrotną od was na temat nowego schematu kolorystycznego: czy wszystko działa prawidłowo, czy wszystko jest czytelne?

![Zrzut ekranu z 2023-07-15 o 13 53 38-fullpage](https://github.com/TomaszGasior/RadioLista-v3/assets/1395027/2a40c591-c95b-4607-ac19-d0d1a71b33b7)

**Comment by HyperDX** [comment]
Bardzo mi się podoba nowy ciemny motyw! Z jednej strony szkoda, że powstał kosztem usunięcia możliwości personalizacji koloru tła wykazu pod siebie, ale ostatnimi czasy i tak kolory dostosowywałem tak, aby nie męczyć oczu :).
